satellite/marketing/marketingweb/server.go

```diff
@@ -48,20 +78,22 @@ func (s *Server) commonPages() []string {
 }
 
 // NewServer creates new instance of offersweb server
-func NewServer(logger *zap.Logger, config Config, listener net.Listener) *Server {
+func NewServer(logger *zap.Logger, config Config, service *marketing.Service, listener net.Listener) *Server {
 	s := &Server{
 		log:      logger,
 		config:   config,
 		listener: listener,
+		service:  service,
 	}
 
 	logger.Sugar().Debugf("Starting Marketing Admin UI on %s...", s.listener.Addr().String())
-
 	fs := http.StripPrefix("/static/", http.FileServer(http.Dir(s.config.StaticDir)))
 	mux := mux.NewRouter()
 	if s.config.StaticDir != "" {
+		mux.HandleFunc("/", s.getOffers)
 		mux.PathPrefix("/static/").Handler(fs)
-		mux.Handle("/", s)
+		mux.HandleFunc("/createFreeCredit", s.createOffer)
```

Comment:
Use more conventional endpoint naming. All lower-case only, with maybe "-"... the internet tends to mangle casing.

Reply:
done